gapis/resolve/state_tree_test.go

```diff
@@ -491,7 +491,7 @@ func TestStateTreeNode(t *testing.T) {
 			&service.StateTreeNode{
 				NumChildren:    10,
 				Name:           "[10 - 19]",
-				ValuePath:      rootPath.Field("ReferenceB").Field("Array").Path(),
+				ValuePath:      rootPath.Field("ReferenceB").Field("Array").Slice(10, 19).Path(),
```

Comment:
Now that these paths are unique, you can probably kill the `flags    int` field from the test structure. The only reason this existed was that there was a many-to-1 mapping of value path -> STN. Now I believe they're all 1:1.